x-pack/test_serverless/shared/lib/security/kibana_roles/project_controller_security_roles.yml

```diff
@@ -504,6 +506,9 @@ soc_manager:
         - feature_siem.execute_operations_all
         - feature_siem.scan_operations_all
         - feature_securitySolutionCases.all
+        - feature_securitySolutionCasesV2.all
+        - feature_observabilityCases.all
+        - feature_observabilityCasesV2.all
```

Comment:
Some tests that use these roles test that they can make use of case features in all plugins, osquery for instance will use one of these roles, and then test that it can attach query results to security cases and observability cases. https://github.com/elastic/kibana/blob/main/x-pack/plugins/osquery/cypress/e2e/all/cases.cy.ts#L33 https://github.com/elastic/kibana/blob/main/x-pack/plugins/osquery/cypress/e2e/all/cases.cy.ts#L64

and so I think the changes in this file should remain.

Reply:
I'm not quite sure why we need both v1 and v2 privileges in this file, cannot we just keep v2 ones?